### PR TITLE
feat(1488): opc-nav position fixed to top in chrome

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Resolves

1488

# Explain the feature/fix

1. Fixed opc-nav not getting position fix to top in chrome
2. Updated readme

## Does this PR introduce a breaking change
No

## Screenshots

<details>
<summary>View Screenshots</summary>

| Description |  Screenshot |
| ----------- | ------------ |
| On first render with overflow |  Screenshot 2021-09-20 at 10 53 22 PM |
| On scroll | Screenshot 2021-09-20 at 10 53 31 PM |

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [x] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
#### If it is a new component
- [x] Does your component follow One Platform style guidelines?
- [x] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
- [x] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
#### Browsers you have tested in
- [x] Latest two versions of Mozilla Firefox and Google Chrome supported by Red Hat Enterprise Linux distribution
- [x] Google Chrome [Latest 2 versions]
- [x] Mozilla Firefox [Latest 2 versions]
- [x] Microsoft Edge [Latest 2 versions]
- [ ] Apple Safari [Latest 2 versions]
- [ ] Microsoft Internet Explorer 11

### CODE REVIEW

**Comment by akhilmhdh** [comment]
Tested in my three browsers and chrome especially @hybridx  😅

**Review by ghanlohar — APPROVED** [review]
LGTM

File: packages/opc-nav/src/opc-nav-search.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+button,
+input {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font-weight: inherit;
+  vertical-align: baseline;
+  background: none;
+}
```
Comment (hybridx) [inline packages/opc-nav/src/opc-nav-search.scss:19]: @gisumit aren't all these changes already available in opc-styles, @akhilmhdh can you check?

**Review by hybridx — CHANGES_REQUESTED** [review]

**Review by gisumit — APPROVED** [review]
LGTM

File: packages/opc-nav/src/opc-nav-search.scss (hunk 1 above)
Comment (gisumit) [reply packages/opc-nav/src/opc-nav-search.scss:19]: The margin & paddings are reset in the opc-styles package.

Comment (akhilmhdh) [reply packages/opc-nav/src/opc-nav-search.scss:19]: margin padding yes but not some of them

**Review by hybridx — APPROVED** [review]
LGTM

**Pull request merged**